src/components/Books.js

```diff
@@ -1,163 +1,76 @@
 import React, { useState } from 'react';
-import './Main.css';
-import './Book.css';
-import { CircularProgressbar, buildStyles } from 'react-circular-progressbar';
+import { useDispatch } from 'react-redux';
+import { addBookAsync } from '../redux/books/booksSlice';
+import BookList from './BooksList';
 import Header from './Header';
-import 'react-circular-progressbar/dist/styles.css';
+import './Main.css';
 
-function Book({
-  title, author, progress, category, onProgressUpdate,
-}) {
-  return (
-    <div className="bookCard">
+// end imports
 
-      <div className="bookInfo">
-        <div className="book">
-          <div className="bookDetails">
-            <span>
-              {' '}
-              {category}
-              {' '}
-              <br />
-              <b>
-                {' '}
-                {title}
-              </b>
-              <br />
-              <p>
-                {' '}
-                {author}
-                {' '}
-              </p>
-            </span>
+const App = () => {
+  const dispatch = useDispatch();
+  const [selectedApp] = useState('bMrmeF1Ldn440EWBDcT5');
 
-            <ul className="list">
-              <li>remove</li>
-              <li>edit</li>
-              <li>delete</li>
-            </ul>
-          </div>
-        </div>
+  const [book, setBook] = useState({
+    title: '',
+    author: '',
+  });
 
-        <div className="progressDiv">
-          <CircularProgressbar
-            className="progress"
-            value={progress}
-            text={`${progress}%`}
-            styles={buildStyles({
-              textSize: '16px',
-              pathColor: `rgba(62, 152, 199, ${progress / 100})`,
-              textColor: '#f88',
-              trailColor: '#d6d6d6',
-            })}
-          />
-        </div>
-        <div className="onPBtn" style={{ display: 'flex', flexDirection: 'column', paddingInlineEnd: '20%' }}>
-          <i>Current Chapter</i>
-          <button type="button" onClick={onProgressUpdate}>Update Progress</button>
-        </div>
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+    setBook({ ...book, [name]: value });
+  };
 
-      </div>
-    </div>
-  );
-}
+  function generateUUID() {
+    return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, (c) => {
+      const r = Math.random() * 16 | 0; const
+        v = c === 'x' ? r : (r & 0x3 | 0x8);
+      return v.toString(16);
+    });
+  }
 
-function useLocalStorage(key, initialValue) {
-  const [storedValue, setStoredValue] = React.useState(() => {
-    try {
-      const item = window.localStorage.getItem(key);
-      return item ? JSON.parse(item) : initialValue;
-    } catch (error) {
-      return initialValue;
-    }
-  });
+  const handleAddBook = async () => {
+    const newBook = {
+      item_id: generateUUID(),
+      title: book.title,
+      author: book.author,
+      category: 'Fiction',
```

Comment:
- [ ] Please make sure that the grey background us properly set in your app. Also make sure that the typography us the same as in the design guideline (font family, size, style, weight etc). See the screenshot for reference:

Your Implementation | Zeplin Design 
----|----
<img width="1231" alt="Screenshot 2023-09-15 at 11 47 57 AM" src="https://github.com/paulshaun-kdtk/React_Book_Store/assets/44798044/46020b26-bb13-487e-ab5e-2c95eec3ad2d">|<img width="611" alt="Screenshot 2023-09-15 at 12 06 29 PM" src="https://github.com/paulshaun-kdtk/React_Book_Store/assets/44798044/0f873636-677f-472a-9c20-5dd0af1cc8c9">